calicoctl/calico_ctl/checksystem.py

```diff
@@ -246,3 +253,37 @@ def _check_etcd_version():
             system_ok = False
 
     return system_ok
+
+
+def _check_kernel_version():
+    """
+    Check that the kernel version meets the minimum required version for Calico.
+    """
+    full_version = platform.uname()[2]
+
+    # We just need the Major, and Minor, and Bug fix version numbers
+    regex = re.compile("([\d]\.[\d]+\.[\d]+)[-+].*")
```

Comment:
I wonder if there is always a - or +?   or can it ever be just a simple version string 1.2.3?